test/integration/tests/import.sh

```diff
@@ -4,8 +4,8 @@ source helpers.sh
 
 cleanup() {
     rm -f import_key.ctx import_key.name import_key.priv import_key.pub \
-    parent.ctx plain.dec.ssl  plain.enc plain.txt sym.key import_rsa_key.pub \
-    import_rsa_key.priv import_rsa_key.ctx import_rsa_key.name private.pem \
+    parent.ctx plain.dec.ssl  plain.enc plain.txt sym.key import_rsa_key*.pub \
```

Comment:
Is this a rebase left over? I think for the feature you are enabling this change doesn't add up. All changes to this file fall in the same camp.